action.yml

```diff
@@ -109,7 +113,7 @@ runs:
           ${{ (inputs.skip-test == 'true' && '--skip-test') || '' }} \
           ${{ (inputs.shuffle == 'true' && '--shuffle on') || '' }} \
           ${{ (inputs.concurrent == 'true' && '--concurrent on') || '' }} \
-          ${{ (inputs.enable-read-remote-scope == 'true' && inputs.enable-run-exec-scope == 'true' && '--scopes read:remote,run:exec') || '' }}  \
-          ${{ (inputs.enable-read-remote-scope == 'true' && inputs.enable-run-exec-scope == 'false' && '--scopes read:remote') || '' }}  \
-          ${{ (inputs.enable-read-remote-scope == 'false' && inputs.enable-run-exec-scope == 'true' && '--scopes run:exec') || '' }}  \
+          ${{ (inputs.enable-read-parent-scope == 'true' && '--scopes read:parent') || '' }}  \
+          ${{ (inputs.enable-read-remote-scope == 'true' && '--scopes read:remote') || '' }}  \
+          ${{ (inputs.enable-run-exec-scope == 'true' && '--scopes run:exec') || '' }}  \
```

Comment:
Since there are more types of SCOPES, I felt it was appropriate to stop specifying options with comma separators for ease of maintenance